changelog.d/63.misc

```diff
@@ -0,0 +1 @@
+Add Dockerfile
```

Comment:
```suggestion
Add Dockerfile.
```

We demand punctuation in Synapse. Not sure if it's the same here but can't hurt :)